README.md

```diff
@@ -12,6 +12,12 @@ It also helps with server-side communication, taming async callbacks with promis
 and it makes client-side navigation and deep linking with hashbang urls or HTML5 pushState a
 piece of cake. Best of all? It makes development fun!
 
+##### AngularJS will be moving to Long Term Support (LTS) mode on July 1st: [Find out more](misc/version-support-status)
+
+##### Looking for the new Angular? Go here: https://github.com/angular/angular
+
+--------------------
```

Comment:
Can you put another rule above AngularJS will be moving ... ? That'll frame it nicely